### PR TITLE
Docs: readme known issues

### PR DESCRIPTION
**Pull request by dylanmedlin**
Adds a "Known Issues" section to the readme file. And includes the fix to an issue that can happen when installing the translator on macOS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ Translates the SplashKit C++ source into another language.
 - [Running](#running)
    - [Docker](#docker)
    - [Dependencies](#dependencies)
+   - [Known Issues](#known-issues)
```
Comment (tancnle) [inline README.md:14]: **suggestion:** What do you think about `Troubleshooting` instead? And also move this under `Converting`. Unexpected issues are normally presented at the end so they would not obstruct the normal flow.

```suggestion
   - [Troubleshooting](#troubleshooting)
```

File: README.md [hunk 2]
```diff
@@ -59,6 +60,16 @@ $ bundle install
 
 Then run using `translate`.
 
+## Known Issues
+
+- ### macOS Bundle Install error
```
Comment (tancnle) [inline README.md:65]: **suggestion:** Header should not be wrapped in a list item.

```suggestion
### macOS Bundle Install error
```

**Review by jacksievers — APPROVED** [review]
Approving this as it is simply just addresses some known issues

**Review by wil-saman — APPROVED** [review]
Approved as it addresses a known issue that others encountered while installing the translator on macOS.

**Review by Xiphias5985 — APPROVED** [review]
Looks good. Adds an important section to the README.

File: README.md [hunk 3]
```diff
@@ -59,6 +60,16 @@ $ bundle install
...
+
+  If you get the following error when running `bundle install`:
+
+  `can't find gem bundler (>= 0.a) with executable bundle (Gem::GemNotFoundException)`
```
Comment (tancnle) [inline README.md:69]: **suggestion:** Can we convert this to multi-line code block?
 ~~~
```plaintext
can't find gem bundler (>= 0.a) with executable bundle (Gem::GemNotFoundException)
```
 ~~~

**Review by tancnle — COMMENTED** [review]
Thank you @dylanmedlin. Sorry for the late review. Since this is already merged, feel free to consider these suggestions in another PR.

**Pull request merged**